Source/WebKit/WebProcess/GPU/media/ios/RemoteMediaSessionHelper.cpp

```diff
@@ -72,8 +72,10 @@ void RemoteMediaSessionHelper::stopMonitoringWirelessRoutesInternal()
     ensureConnection().send(Messages::RemoteMediaSessionHelperProxy::StopMonitoringWirelessRoutes(), { });
 }
 
-void RemoteMediaSessionHelper::providePresentingApplicationPID(int pid)
+void RemoteMediaSessionHelper::providePresentingApplicationPID(int pid, ShouldOverride shouldOverride)
 {
+    UNUSED_PARAM(shouldOverride);
+    ASSERT(shouldOverride == ShouldOverride::No);
```

Comment:
This can be `ASSERT_UNUSED(shouldOverride, shouldOverride == ShouldOverride::No);`

Reply:
Will fix.